pkg/api/configsync/v1alpha1/resource_override.go

```diff
@@ -68,8 +68,8 @@ type OverrideSpec struct {
 	EnableShellInRendering *bool `json:"enableShellInRendering,omitempty"`
 }
 
-// ContainerResourcesSpec allows to override the resource requirements for a container
-type ContainerResourcesSpec struct {
+// ContainerSpec allows to override the resource requirements for a container
+type ContainerSpec struct {
```

Comment:
must've missed this one

Reply:
probably don't need to add new fields to the old API. Just add it to v1beta1

Reply:
removed from v1alpha1, only declared in v1beta1

Reply:
When adding new fields, it needs to be added to both api versions so that api conversions work. We need to continue doing that until we remove v1alpha1.

Reply:
I don't think this is actually true. The conversions use the v1beta1 as the storage version. So they convert from v1alpha1 -> v1beta1, which is not lossy. There shouldn't be any use case that requires round trip the other way.